aws/src/test/java/org/apache/iceberg/aws/s3/signer/TestS3SignRequestParser.java

```diff
@@ -126,7 +126,7 @@ public void roundTripSerde() {
                     "Content-Type",
                     Arrays.asList("application/json"),
                     "User-Agent",
-                    Arrays.asList("aws-sdk-java/2.17.257", "Linux/5.4.0-126")))
+                    Arrays.asList("aws-sdk-java/2.20.16", "Linux/5.4.0-126")))
```

Comment:
updating this isn't really necessary. When I wrote the test I just wanted to have some real headers :)

We can also just update this to have some random number if that's confusing people.

Reply:
I think it's perfectly fine :), In my case I just greped the existing version of sdk and replaced it !